KeyboardShortcutsExample/ContentView.swift

```diff
@@ -6,42 +6,57 @@ extension KeyboardShortcuts.Name {
 	static let testShortcut2 = Self("testShortcut2")
 }
 
+struct ShortCut {
+	let name: KeyboardShortcuts.Name
+	let id: Int8
+	let label: String
+}
+
 struct ContentView: View {
-	@State private var isPressed1 = false
-	@State private var isPressed2 = false
+	@State private var isPressed = false
+	@State private var selectedState = 0
+	var shortCuts = [
+		ShortCut(name: .testShortcut1, id: 0, label: "ShortCut1"),
```

Comment:
No need for `id`. Can just use the array index for that.

Reply:
> It's `Shortcut`, not `ShortCut` (applies everywhere).

OK! Sorry for my bad English 😢